src/drawing.rs

```diff
@@ -22,6 +25,8 @@ impl Drawing {
             canvas_transform: Transformation::new(),
             elements: Vec::new(),
             attrs: HashMap::new(),
+            refdes: None,
```

Comment:
The original idea was to add `ref_des` and `value` to `elements` vector as the `Drawing` struct is intended to be used both for Symbol and Footprint. Need to discuss whether it was a good idea or not.

Reply:
My first thought was that refdes and value are important things and deserve their own fields. But your idea of putting them into `elements` came to my mind too. It may require a bit different logic inside `kicad` generator, but seems like a better design choice for now.